src/Snowcap/Emarsys/Client.php

```diff
@@ -418,6 +418,21 @@ public function getContactsFromContactList($listId, array $data)
     }
 
     /**
+     * Checks whether a specific contact is included in the defined contact list.
```

Comment:
Please change "Checks" into "Check" and remove trailing perion in order to march coding style of a project.